curation-migration-datasync/externalCaller/parser.spec.ts

```diff
@@ -1,21 +1,22 @@
 import nock from 'nock';
 import { getParserMetadata } from './parser';
-import config from './config';
+import config from '../config';
 
 describe('getParsedDomainId', () => {
   it('fetches appropriate keys from response', async () => {
-    const data = { resolved_id: '1', item: { domain_id: '124' } };
+    const data = { resolved_id: 1, item: { domain_id: '124' } };
```

Comment:
I'm not sure this should be changed -- when I query the parser endpoint, the resolvedId is a string

Reply:
that is true

Reply:
thanks for catching this, i was making it closer to db datatype but realised the parser returns them as string, changed it back to string in other places but this one got left out

fixed now